apps/backoffice-v2/src/lib/blocks/hooks/useDocumentBlocks/hooks/useCommentInput/useCommentInput.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback, useState } from 'react';
+
+export const useCommentInput = () => {
+  const [value, setValue] = useState<string | undefined>(undefined);
+
+  const handleChange = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
These are just in general the same comment. I think the names should reflect its for comment.
```suggestion
  const onCommentChange = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
```